python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -397,6 +397,11 @@ def _annotate_image(fig, source, flags):
     if any(flags[flags_shape]):
         fig.text(0.87, 0.83, "SHAPE", color="red", fontweight="bold")
 
+    # rb score
+    fig.text(0.73, 0.79, f"RB:{source['spuriousness']:.03f}",
+             color="red" if source['spuriousness'] < 0.5 else "green",
```

Comment:
Although the colors for the text above are just to help make each flag pop out (the actual colors themselves matter much less), this is a case where red/green colorblindness is a problem. Maybe "red" and "blue" instead?

Reply:
I am replacing the colors here per your suggestion. But in general, I prefer such concepts to be captured and implemented in a more structured way, not so randomly. E.g. I do have my own color-blind-support colormap when generating figures for my papers. Don't we have such a thing implemented for LSST already?